commit c7f7f86126e43399130b021bac07c60b2f0e4892 (refs/changes/30/330/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-08-22T10:45:21-07:00 (3 years, 2 months ago)
    
    TOOLS-1531 node-triton README should have the "use gerrit" blurb

diff --git a/README.md b/README.md
index b0b8f07..bfe2eee 100644
--- a/README.md
+++ b/README.md
@@ -2,6 +2,11 @@
 
 # node-triton
 
+This repository is part of the Joyent Triton project. See the [contribution
+guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
+*Triton does not use GitHub PRs* -- and general documentation at the main
+[Triton project](https://github.com/joyent/triton) page.
+
 `triton` is a CLI tool for working with the CloudAPI for Joyent's Triton [Public Cloud]
 (https://docs.joyent.com/public-cloud) and [Private Cloud] (https://docs.joyent.com/private-cloud).
 CloudAPI is a RESTful API for end users of the cloud to manage their accounts, instances,
@@ -14,13 +19,11 @@ for CloudAPI. `node-smartdc` CLI works off the 32-character object UUID to uniqu
 identify object instances in API requests, and returns response payload in JSON format.
 The CLI covers both basic and advanced usage of [CloudAPI](https://apidocs.joyent.com/cloudapi/).
 
-As a lightweight programmable interface for CloudAPI, the `triton` CLI supports both name or
-UUID identification of object instances and the use of short ID, as well as the choice
-between concise tabular responses and full JSON responses.
-
-**The `triton` CLI is currently in beta and will be expanded over time to
-support all CloudAPI commands, eventually replacing `node-smartdc` as both the
-API client library for Triton cloud and the command line tool.**
+**The `triton` CLI is currently in beta (effectively because it does not yet
+have *complete* coverage of all commands from node-smartdc) and will be
+expanded over time to support all CloudAPI commands, eventually replacing
+`node-smartdc` as both the API client library for Triton cloud and the command
+line tool.**
 
 ## Setup
 
